tests/Sdk/Unit/Trace/SpanProcessor/BatchSpanProcessorTest.php

```diff
@@ -178,22 +179,14 @@ public function shutdownCallsExporterShutdown()
     public function noExportAfterShutdown()
     {
         $exporter = self::createMock(Exporter::class);
-        // grap a reference to the InvocationOrder object so we can inspect/assert *when* the call happens
-        $exporter->expects($spy = $this->exactly(1))->method('shutdown');
-
-        $this->assertEquals(0, $spy->getInvocationCount());
+        $exporter->expects($this->exactly(1))->method('shutdown');
```

Comment:
This isn't required at all, just by saying 
```php
$this->exactly(1);
```

is equivalent to the checks removed, if anything else happens, an exception is thrown.

These were removed because `getInvocationCount()` is marked internal by PHPUnit, not for use by collaborators.

Reply:
Seems reasonable.